src/functions-and-arrays.js

```diff
@@ -1,19 +1,35 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 < num2) return num2
+   else 
+   return num1
+  }
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(arr) {
+  let longestWord = arr[0]
+for (let words of arr) {
+  if (words.length > longestWord.length) {
+    longestWord = words
+  }
+}
+return longestWord;
+}
```

Comment:
There is one test case that expect you to return null when the array is empty. This you can do by a separate if-statement at the beginning of the function `if (arr.length === 0) return null;`.